plugins/block-shareable-procedures/src/blocks.ts

```diff
@@ -1202,6 +1202,14 @@ const procedureCallerOnChangeMixin = {
   onchange: function (event) {
     if (this.disposed || this.workspace.isFlyout) return;
     if (event.type === Blockly.Events.BLOCK_MOVE) this.updateArgsMap_(true);
+    if (
+      event.type === Blockly.Events.BLOCK_CHANGE &&
+      event.blockId === this.id &&
+      event.element === 'disabled' &&
+      event.group
+    ) {
+      this.previousEnabledState_ = !event.newValue;
```

Comment:
Nit: This is equivalent but I think it's a little clearer.
```suggestion
      this.previousEnabledState_ = event.oldValue;
```

Reply:
Wanted to come back here and say that as I was rewriting this, you're right that `!event.newValue` is the correct way to do this :P